pkg/client/resmgmt/example_test.go

```diff
@@ -316,7 +317,7 @@ func ExampleClient_UpgradeCC() {
 	}
 
 	ccPolicy := cauthdsl.SignedByMspMember("Org1MSP")
-	req := UpgradeCCRequest{Name: "ExampleCC", Version: "v1", Path: "path", Policy: ccPolicy}
+	req := UpgradeCCRequest{Name: "ExampleCC", Version: "v1", Lang: pb.ChaincodeSpec_GOLANG, Path: "path", Policy: ccPolicy}
```

Comment:
Should revert this, as this PR doesn't break the old client code.

Reply:
Thanks for pointing out, that example_test.go has been reverted.